samples/LocalizationPlayground/Properties/launchSettings.json

```diff
@@ -0,0 +1,8 @@
+{
+  "profiles": {
+    "LocalizationPlayground": {
+      "commandName": "Project",
+      "commandLineArgs": "[env:DOTNET_SYSTEM_GLOBALIZATION_CULTURE=de-DE] -c 3 .NET"
```

Comment:
This env should not be needed

Reply:
There should be a .net core leveled env to do the same thing

Reply:
@wli3  We were hoping to align to whatever environment variables exist already in the SDK but the only one I could find was for specifying the invariant culture, not for specifying other cultures. https://docs.microsoft.com/en-us/dotnet/core/run-time-config/globalization

Reply:
my memory is wrong. https://github.com/dotnet/sdk/blob/b1223209644d900702287faea8e9b71f95ec49f8/src/Cli/dotnet/UILanguageOverride.cs#L38 

we had a CLI specific flag. If we need a flag for this library that's fine.